src/convertToObject.js

```diff
@@ -6,7 +6,21 @@
  * @return {object}
  */
 function convertToObject(sourceString) {
-  // write your code here
+  const styles = {};
+  const declarations = sourceString
+    .split(';')
+    .map((declaration) => declaration.trim())
+    .filter(Boolean);
```

Comment:
[CODE STYLE] - All chained methods called on the same level should have the same indentation. This makes code much more readable. Ensure that the methods `.split`, `.map`, and `.filter` are aligned to improve readability.